[Interactive Graph] Ensure we have defaults for labels in our Locked Figures (#3765)
Khan/perseus: .changeset/thirty-cheetahs-know.md

---
"@khanacademy/perseus-core": patch
---

> Default addition to parser and editor for labels in locked figures

Reviewer: This might be me, but I'm having trouble understanding this changelog. What do you think of something like this? 

```suggestion
Provide defaults in the parser for labels in locked figures
```

Author: I agree with the change, I think it adds more clarity
